PencilCase/PencilCase/scripts/CustomBindings/InitialSelectedTabBinding.ts

```diff
@@ -1,5 +1,5 @@
 ﻿class InitialSelectedTabBinding implements KnockoutBindingHandler {
-    public update(element: any, valueAccessor: () => any, allowBindingAccessor: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: KnockoutBindingContext) {
+    public init(element: any, valueAccessor: () => any, allowBindingAccessor: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: KnockoutBindingContext) {
```

Comment:
Here use init is good enough. No need to use update because we only need to set the subscription one time when initializing.